Recon/python/recoSkim3.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
+
+import sys
+import os
+import json
+
+# we need the ldmx configuration package to construct the object
+
+from LDMX.Framework import ldmxcfg
+
+# first, we define the process, which must have a name which identifies this
+# processing pass ("pass name").
+# the other two pass names refer to those of the input files which will be combined in this job.
+
+thisPassName="reconSeq" #overlay" 
+nElectrons=1
+p=ldmxcfg.Process( thisPassName )
+
+#import all processors
+
+# Ecal hardwired/geometry stuff
+from LDMX.Ecal import EcalGeometry
+import LDMX.Ecal.ecal_hardcoded_conditions
+egeom = EcalGeometry.EcalGeometryProvider.getInstance()
```

Comment:
If you don't need the geometry handle to change any of its paramters, this line is not needed and is just clutter.

```suggestion
```

Reply:
Fixed

Reply:
(in comments of issue)